Bucketeer/Sources/Internal/DI/DataModule.swift

```diff
@@ -4,6 +4,8 @@ protocol DataModule {
     var config: BKTConfig { get }
     var userHolder: UserHolder { get }
     var apiClient: ApiClient { get }
+    var evaluationUserDefaultsDao: EvaluationUserDefaultsDao { get }
+    var evaluationStorage: EvaluationStorage { get }
     var evaluationDao: EvaluationDao { get }
     var eventDao: EventDao { get }
     var defaults: Defaults { get }
```

Comment:
Can you remove the following fields from the protocol definition?

- `evaluationUserDefaultsDao`
- `evaluationDao`
- `defaults`

Now that there is an `EvaluationStorage` class, we should restrict the visibility of DAO and other implementation details.

Reply:
thank @yshrsmz for reviewing 
let me try to remove them.